appveyor.yml

```diff
@@ -24,27 +24,23 @@ environment:
 # Adapt as necessary starting from here
 
   matrix:
-  # - R_VERSION: devel
```

Comment:
@drnickisaac Did you mean to make these changes? This is reversing my fix to Appveyor

Reply:
Oops. No, I did not mean to do this. I suspect it occurred when I pulled the upstream master. Is the solution to simply add this line back in?

Reply:
Ah - I see. I did not commit appveyor.yml after pulling the upstream master. I have done this now and the other issue. You'll see the revised PR very soon.